frontend/src/credit_transfers/components/CreditTransferTable.js

```diff
@@ -94,7 +94,8 @@ const CreditTransferTable = (props) => {
     id: 'status',
     Header: 'Status',
     accessor: item => ((item.status.id === CREDIT_TRANSFER_STATUS.completed.id)
-      ? CREDIT_TRANSFER_STATUS.approved.description : item.status.status),
+      ? CREDIT_TRANSFER_STATUS.approved.description : (Object.entries(CREDIT_TRANSFER_STATUS)
```

Comment:
You can get rid of the ternary check and just replace the description for completed to "Approved" to simplify the statement here